src/button/component.jsx

```diff
@@ -721,6 +751,13 @@ export let Button : Component<ButtonOptions> = create({
                             [FPTI.KEY.BUTTON_SESSION_UID]: this.props.buttonSessionID
                         });
                     }
+                    
+                    if (fundingLogoThrottle.isActive()) {
```

Comment:
Would also recommend adding a `logComplete()` in onAuthorize.

Reply:
Done.